src/components/ControlledSelectInputField/ControlledSelectInputField.js

```diff
@@ -0,0 +1,80 @@
+import { makeStyles, useTheme } from "@material-ui/core"
+import { useMemo } from "react"
+import { Controller } from "react-hook-form"
+import Select from "react-select"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    fontSize: theme.typography.fontSize,
+    fontFamily: theme.typography.fontFamily,
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightBold,
+    paddingBottom: "4px",
+  },
+}))
+
+const ControlledSelectInputField = ({
+  control,
+  name,
+  label,
+  placeholder,
+  options,
+}) => {
+  const classes = useStyles()
+  const theme = useTheme()
+
+  const customStyles = useMemo(() => {
+    return {
+      control: (baseStyles) => ({
+        ...baseStyles,
+        minHeight: 0,
+        height: "30px",
+      }),
+      valueContainer: (baseStyles) => ({
+        ...baseStyles,
+        padding: "2px 8px 8px 8px",
+      }),
+      dropdownIndicator: (baseStyles) => ({
+        ...baseStyles,
+        padding: "2px 8px 8px 8px",
+      }),
+      indicatorSeparator: (baseStyles) => ({
+        ...baseStyles,
+        marginTop: 0,
+        marginBottom: "5px",
+      }),
+      placeholder: (baseStyles) => ({
+        ...baseStyles,
+        color: theme.palette.text.hint,
+      }),
+    }
+  }, [theme])
+
+  return (
+    <div className={classes.root}>
+      <span className={classes.label}>{label}</span>
+
+      <Controller
+        control={control}
+        name={name}
+        render={({ value, onChange }) => {
+          return (
+            <Select
+              styles={customStyles}
+              options={options}
+              defaultValue={null}
```

Comment:
Please remove this prop, the select will only get it's value from the `value` prop